dolphinscheduler-ui/src/views/resource/udf/resource/use-table.ts

```diff
@@ -285,18 +298,19 @@ export function useTable() {
     router.push({ name: 'resource-manage' })
   }
 
-  const goBread = (fullName: string) => {
-    const { id } = variables
+  const goBread = (fileName: string) => {
+    const id = -1
```

Comment:
If `id` is useless, it is better to remove it.

Reply:
ya, the "id" is deprecated. Will remove it in another pr along with all other "ids". Right now they are labeled as Deprecated in the backend